### ISSUE TITLE
SyntaxError: Unexpected end of JSON input - promise_server.js:218

### ISSUE THREAD

**Issue by sabedinia** [meteor/meteor#12148]
**Here is how you can reproduce the issue**

step 1. meteor create --full myproject --release 2.3
step 2. add either "pdfmake": "^0.2.5" or "node-sass": "^7.0.1" to package.json
step 3. meteor npm install
step 4. meteor

you should get this:

=> Started proxy.                             
/.meteor/packages/meteor-tool/.2.3.1.gbhcyh.f17sq++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:218
      throw error;
      ^

SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)

Running on Mac OS M1 on rosetta

**Comment by willosof** [comment]
I'm experiencing this too. 

I've tried `meteor reset`, and this is my package.json:
https://gist.github.com/willosof/1269c02d9f27e470f76046e670f1c45c

```
❯ uname -a
Darwin gramac 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64 x86_64

❯ node -v
v16.17.0
```

**Comment by doyskie7** [comment]
I have this experience when i installed node-sass, when i removed it, it worked properly, but i need sass compiler on our meteor app, I tried [fourseven](fourseven:scss) but still have an issue reading our scss file.

**Comment by AlexanderOlivares** [comment]
For those still encountering this issue -- according to #11839 the fix is upgrading to v2.5.3 https://github.com/meteor/meteor/issues/11839#issuecomment-1007391404
